core/packages/test/scripts/detect-fork.ts

```diff
@@ -118,24 +118,12 @@ const NONCES = [
     storageKey:
       "0x684b82bef882079feeabe54a5bd7b94a718368a0ace36e2b1b8b6dbd7f8093c0",
   },
-  {
```

Comment:
This file can probably deleted completely. It was made to handle hardforks in the ethereum 1 light client which we do not use anymore.